examples/analytics.amp.html

```diff
@@ -33,11 +34,22 @@
   "triggers": {
     "defaultPageview": {
       "on": "visible",
-      "request": "event",
+      "request": "pageview",
       "vars": {
         "eventName": "page-loaded",
         "eventId": "42"
       }
+    },
+    "scrollPings": {
+      "on": "scroll",
+      "request": "event",
+      "scroll-spec": {
+        "vertical-boundaries" : [1, 50, 90],
```

Comment:
@rudygalfi If you know what the usecase is, can you open another issue for it, cc people who need the feature and also assign other labels?